package.json

```diff
@@ -1,60 +1,90 @@
-    "fields"
+    "sanity-plugin"
   ],
-  "author": "ahm digital (Cam Chamberlain)",
-  "license": "MIT",
+  "homepage": "https://github.com/ahmdigital/fieldwork#readme",
   "bugs": {
     "url": "https://github.com/ahmdigital/fieldwork/issues"
   },
+  "repository": {
+    "type": "git",
+    "url": "git+ssh://git@github.com/ahmdigital/fieldwork.git"
+  },
+  "license": "MIT",
+  "author": "ahm digital <joshua.edwards@medibank.com.au>",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "source": "./src/index.ts",
+      "import": "./dist/index.esm.js",
+      "require": "./dist/index.js",
+      "default": "./dist/index.esm.js"
+    },
+    "./package.json": "./package.json"
+  },
+  "main": "./dist/index.js",
+  "module": "./dist/index.esm.js",
+  "source": "./src/index.ts",
+  "types": "./dist/index.d.ts",
+  "files": [
+    "dist",
+    "sanity.json",
+    "src",
+    "v2-incompatible.js"
+  ],
+  "scripts": {
+    "build": "run-s clean && plugin-kit verify-package --silent && pkg-utils build --strict && pkg-utils --strict",
+    "clean": "rimraf dist",
+    "format": "prettier --write --cache --ignore-unknown .",
+    "link-watch": "plugin-kit link-watch",
+    "lint": "eslint .",
+    "prepublishOnly": "run-s build",
+    "test": "jest",
+    "watch": "pkg-utils watch --strict"
+  },
   "dependencies": {
+    "@sanity/incompatible-plugin": "^1.0.4",
     "lodash": "^4.17.19",
-    "prop-types": "^15.7.2",
-    "react": "^16.13.1",
-    "react-icons": "^3.10.0",
-    "styled-components": "^5.1.0"
-  },
-  "peerDependencies": {
-    "@sanity/base": "^2.0.0"
+    "react-icons": "5.0.1"
   },
   "devDependencies": {
-    "@ahmdigital/eslint-config": "7.1.40",
-    "@typescript-eslint/parser": "6.18.1",
-    "babel-cli": "6.26.0",
-    "babel-preset-es2015": "6.24.1",
-    "babel-preset-react": "6.24.1",
-    "eslint": "8.56.0"
-  },
-  "renovate": {
-    "extends": [
-      "@ahmdigital"
-    ]
+    "@ahmdigital/eslint-config": "7.1.46",
+    "@sanity/pkg-utils": "^4.1.2",
+    "@sanity/plugin-kit": "^3.1.10",
+    "@types/jest": "^29.5.12",
+    "@types/react": "^18.2.54",
+    "@types/styled-components": "^5.1.34",
+    "@typescript-eslint/eslint-plugin": "^6.21.0",
+    "@typescript-eslint/parser": "^6.21.0",
+    "eslint": "^8.56.0",
+    "eslint-config-prettier": "^9.1.0",
+    "eslint-config-sanity": "^7.0.1",
+    "eslint-plugin-prettier": "^5.1.3",
+    "eslint-plugin-react": "^7.33.2",
+    "eslint-plugin-react-hooks": "^4.6.0",
+    "jest": "^29.7.0",
+    "npm-run-all": "^4.1.5",
+    "prettier": "^3.2.5",
+    "prettier-plugin-packagejson": "^2.4.10",
+    "react": "^18.2.0",
+    "react-dom": "^18.2.0",
+    "react-is": "^18.2.0",
+    "rimraf": "^5.0.5",
+    "sanity": "^3.27.1",
+    "styled-components": "^5.3.11",
+    "ts-jest": "^29.1.2",
+    "typescript": "^5.3.3"
+  },
+  "peerDependencies": {
+    "react": "^18",
+    "sanity": "^3"
   },
   "engines": {
-    "node": "^16"
-  }
+    "node": ">=14"
```

Comment:
should we consider >=16 or something?

Reply:
To be in line with the rest of our V3 changes, yes. This was generated by the plugin-kit. I've bumped to 16 now.